packs/data/backgrounds.db/animal-wrangler.json

```diff
@@ -0,0 +1,86 @@
+{
+    "_id": "K35I1WCbzT5xnJ6N",
+    "data": {
+        "boosts": {
+            "0": {
+                "value": [
+                    "str",
+                    "wis"
+                ]
+            },
+            "1": {
+                "value": [
+                    "cha",
+                    "con",
+                    "dex",
+                    "int",
+                    "str",
+                    "wis"
+                ]
+            }
+        },
+        "description": {
+            "value": "<p>You have a long history of working with large circus animals for the Celestial Menagerie, such as performing in an animal act, jostling animals back into their cages, or even sweeping dung out of squalid pens. Mistress Dusklight's ongoing mistreatment of her animals compelled you to quit, and you now strive to ensure animals aren't abused.</p>\n<p>Choose two ability boosts. One must be to <strong>Strength</strong> or <strong>Wisdom</strong>, and one is a free ability boost.</p>\n<p>You're trained in your choice of the Athletics or Nature skills. You gain a skill feat: @Compendium[pf2e.feats-srd.Titan Wrestler]{Titan Wrestler} if you chose Athletics, or @Compendium[pf2e.feats-srd.Train Animal]{Train Animal} if you chose Nature. You are also trained in a Lore skill related to a particular kind of common animal (such as Equine Lore, Feline Lore, or Pachyderm Lore).</p>"
+        },
+        "items": {},
+        "rules": [
+            {
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "PF2E.SkillAth",
+                        "value": "athletics"
+                    },
+                    {
+                        "label": "PF2E.SkillNat",
+                        "value": "nature"
+                    }
+                ],
+                "flag": "skillChoice",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.Skill",
+                "rollOption": "animal-wrangler"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "upgrade",
+                "path": "data.skills.{item|flags.pf2e.rulesSelections.skillChoice}.rank",
```

Comment:
```suggestion
                "path": "data.skills.{item|flags.pf2e.rulesSelections.skill}.rank",
```

Reply:
Fixed in [e9a9f35](https://github.com/foundryvtt/pf2e/pull/2807/commits/e9a9f35d268ff5f4dea5061d2282eae1678e3dd3)